File: update_app.py
with open("crm/src/App.jsx", "r", encoding="utf-8") as f:
    text = f.read()

if "import Categories from './pages/Categories';" not in text:
    text = text.replace("import Staff from './pages/Staff';", "import Staff from './pages/Staff';\nimport Categories from './pages/Categories';")
    
if "<Route path=\"equipe\" element={<Staff />} />" in text:
    text = text.replace("<Route path=\"equipe\" element={<Staff />} />", "<Route path=\"equipe\" element={<Staff />} />\n          <Route path=\"categorias\" element={<Categories />} />")

with open("crm/src/App.jsx", "w", encoding="utf-8") as f:
    f.write(text)

File: update_app.js
const fs = require('fs');
let code = fs.readFileSync('crm/src/App.jsx', 'utf8');

code = code.replace(/import GeradorRelatorios from '\.\/pages\/GeradorRelatorios';/, `import CentralRelatorios from './pages/CentralRelatorios';`);
code = code.replace(/<GeradorRelatorios \/>/, `<CentralRelatorios />`);

fs.writeFileSync('crm/src/App.jsx', code, 'utf8');
console.log('App.jsx updated');
